audbackend/__init__.py

```diff
@@ -3,12 +3,17 @@
 from audbackend.core.api import create
 from audbackend.core.api import delete
 from audbackend.core.api import register
-from audbackend.core.artifactory import Artifactory
 from audbackend.core.backend import Backend
 from audbackend.core.errors import BackendError
 from audbackend.core.filesystem import FileSystem
 from audbackend.core.repository import Repository
 
+# Import optional backends
+try:
+    from audbackend.core.artifactory import Artifactory
```

Comment:
I was wondering how will know in the code which backends the user wants to use. so the solution here is to make `Artifactory` visible as soon as `dohq-artifactory` is found in the environment. Sounds like a reasonable solution to me.

Reply:
Yes, the idea is that you can basically add backends by just installing the required dependencies.